src/index.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { render } from 'react-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import '../public/styles/index.scss';
-
```

Comment:
Why remove this? If we remove this, add a line after the BS import.

Reply:
Fixed this